README.md

```diff
@@ -107,11 +108,11 @@ This code was largely taken from the example drops repo, but was also largely cu
 * The "require" section was stripped down to just:
 
 ```yaml
-    "php": ">=7.0.8",
-    "pantheon-systems/example-drops-8-composer": "^2.3",
     "united-philanthropy-forum/km_collaborative": "*"
 ```
 
+* The "*" above was later set to more restricted targeting of releases of km_collaborative.
```

Comment:
I think it's fine to just say "united-philanthropy-forum/km_collaborative": "^0.1.4" above and leave off this comment.

Reply:
Sweet, and updated to match the next comment too ;)